helm_deploy/probation-offender-search/templates/_envs.tpl

```diff
@@ -13,24 +13,9 @@ env:
   - name: SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI
     value: "{{ .Values.env.SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWK_SET_URI }}"
 
-  - name: OAUTH_ENDPOINT_URL
```

Comment:
since we just pass the token straight through, we no longer have a dependency on hmpps-auth